src/popup/components/form/FormTextarea.vue

```diff
@@ -140,7 +140,7 @@ export default defineComponent({
     --size: 8;
   }
 
-  &.auto-height {
+  &.auto-height, &.xxs {
```

Comment:
Why not use the `autoHeight` property instead?

Reply:
Because then it couldn't be resizable
`:class="{ resizable: resizable && !autoHeight }"`